OtripleS.Web.Api/Brokers/Storages/IStorageBroker.StudentRegistrations.cs

```diff
@@ -0,0 +1,12 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+namespace OtripleS.Web.Api.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        
+    }
```

Comment:
We need the interface methods otherwise they won't be available for mocking

Reply:
@hassanhabib my bad, thank you. I will resolve it later